test/data/json/libraries/codec/Lib_EIP155Tx.test.json

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
Using this test file as the source of these tests: https://github.com/ethereumjs/ethereumjs-tx/blob/master/test/ttTransactionTestEip155VitaliksTests.json

Reply:
I didn't test `hash` here, but `sender` implicitly tests this under the hood so it should be fine.

Reply:
Covers tests for `decode`, `encode`, and `sender`.